boards/ambiq/apollo4p_blue_kxr_evb/apollo4p_blue_kxr_evb.dts

```diff
@@ -45,13 +46,18 @@
 
 	buttons {
 		compatible = "gpio-keys";
+		polling-mode;
 		button0: button_0 {
 			gpios = <&gpio0_31 17 (GPIO_PULL_UP | GPIO_ACTIVE_LOW)>;
 			label = "BTN0";
+			zephyr,code = <INPUT_KEY_0>;
+			status = "okay";
 		};
 		button1: button_1 {
 			gpios = <&gpio0_31 19 (GPIO_PULL_UP | GPIO_ACTIVE_LOW)>;
 			label = "BTN1";
+			zephyr,code = <INPUT_KEY_1>;
+			status = "okay";
```

Comment:
The button changes should be in a separate commit, like all other board changes. Actually the commit sequence should look something like this: add input support to the boards, pull new manifest, dts changes, driver implementation commit, board changes to enable usb device controller.

Reply:
Hi @jfischer-no, the commit is now split with the suggested manner and sequence.

Reply:
In the second commit, there are changes to the boards to use drivers that will be introduced in the following commit, which is not what I asked for.

Reply:
Hi @jfischer-no, looks like I did not understand correctly the intent of the original comments. 
I have changed the commit sequence to the following: 

1. Add user button to the boards
2. DTS binding
3. Driver implementation 
4. Board changes to enable USB device controller